common/reducers/wallet.js

```diff
@@ -8,20 +8,30 @@ import type {
 import { BaseWallet } from 'libs/wallet';
 import { toUnit } from 'libs/units';
 import Big from 'bignumber.js';
+import { getTxFromBroadcastStatusTransactions } from 'selectors/wallet';
+
+export type BroadcastStatusTransaction = {
+  isBroadcasting: boolean,
+  signedTx: string,
+  successfullyBroadcast: boolean
+};
```

Comment:
Any reason you didn't want to put this in `libs/transaction` with the rest of 'em?

Reply:
No, good idea, I'll do that.